### ISSUE TITLE
Loss value appears NAN

### ISSUE THREAD

**Issue by qiqiing** [sczhou/IGNN#12]
Hello, when I use the cross-scale attention module in your code and my own data for training, the Loss value becomes NAN as the number of training iterations increases. I can make sure that there is no NAN data in my data. When I reduce the size of the image for training, the training loss becomes normal. Is it because the cross-scale attention module may cause gradient explosion? How can I improve it?

**Comment by qiqiing** [comment]
> Hello, when I use the cross-scale attention module in your code and my own data for training, the Loss value becomes NAN as the number of training iterations increases. I can make sure that there is no NAN data in my data. When I reduce the size of the image for training, the training loss becomes normal. Is it because the cross-scale attention module may cause gradient explosion? How can I improve it?
